src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -619,19 +619,36 @@ function informationDiagram(publicAPI, model) {
     // Add a text label.
     const groupText = group
       .append('text')
-      .attr('x', 6)
+      // .attr('x', 6) // prevents ie11 from seeing text-anchor and startOffset.
       .attr('dy', 15);
 
-    groupText
+    // add a straight path so IE/Edge can measure text lengths usefully.
+    // Otherwise, along a curved path, they return the horizontal space covered.
```

Comment:
Yes, makes sense thanks to your comment.  If you have already tested all this with Chrome and Firefox, then I won't bother with it.  Otherwise, I will do that before we merge.